packages/rspack/module.d.ts

```diff
@@ -9,6 +9,8 @@
  */
 
 declare namespace Rspack {
+	type ModuleId = string | number;
```

Comment:
to double check. in Webpack the moduleId is `string | number`. I assume the number comes from the concatenated modules. do we have the same behavior in rspack?

Reply:
Yes, Rspack's moduleId should be the same as webpack